ccd-definition/AuthorisationCaseField.json

```diff
@@ -825,6 +846,27 @@
     "UserRole": "caseworker-publiclaw-cafcass",
     "CRUD": "R"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "applicant",
+    "UserRole": "caseworker-publiclaw-cafcass",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "applicants",
+    "UserRole": "caseworker-publiclaw-cafcass",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "applicantsMigrated",
```

Comment:
Flag is used only by solicitors. No need to give access to it for other roles. Please remove.

Reply:
They need to be able to see this as it tells them which one is shown.

Reply:
👍

Reply:
👍